src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputWhitespaceAfter.java

```diff
@@ -0,0 +1,62 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace;
+
+public class InputWhitespaceAfter {
```

Comment:
please double all test blocks(valid and with violation) to make new blocks as single line. 
Example:  "do { smth(); } while (condition)".